.github/workflows/create_additional_release_tags.yaml

```diff
@@ -27,10 +27,17 @@ jobs:
 
     - name: Create additional tags
       run: |
+        REF=${{ github.event.release.tag_name || github.event.inputs.releaseTag }}
         ARTIFACT_IDS=('google-cloud-shared-dependencies' 'api-common' 'gax')
         for ARTIFACT_ID in "${ARTIFACT_IDS[@]}"
         do
-          VERSION=$(grep "${ARTIFACT_ID}" versions.txt | cut -d':' -f2)
-          git tag ${ARTIFACT_ID}/$VERSION ${{ github.event.inputs.releaseTag }}
-          git push origin ${ARTIFACT_ID}/$VERSION
+          VERSION=$(grep "^${ARTIFACT_ID}:" versions.txt | cut -d':' -f2 | tr -d '[:space:]')
+          TEMP_BRANCH="temp_branch_${ARTIFACT_ID}_$VERSION"
+          git checkout -b $TEMP_BRANCH
+          git rm .github/workflows/create_additional_release_tag.yaml
+          git commit -m "Remove workflow file"
+          git tag "${ARTIFACT_ID}/$VERSION"
```

Comment:
Can you name tag with "v"?

(It follows convention. https://github.com/googleapis/google-cloud-go/tags)